src/views/arrangement/arrangement.jsx

```diff
@@ -31,14 +32,23 @@ export class Arrange extends Component {
         this.setState({activeTab: active})
     }
 
-    handleTabAdd() {
+    handleTabAdd = () => {
```

Comment:
is this ES6  fat arrow  ?

Reply:
yes

Reply:
Cool. so you keep the context, and it allows you define the functions

Reply:
nice!